openesdh-repo/src/main/java/dk/openesdh/repo/webscripts/groups/Members.java

```diff
@@ -0,0 +1,159 @@
+package dk.openesdh.repo.webscripts.groups;
+
+import org.alfresco.repo.domain.permissions.Authority;
+import org.alfresco.service.cmr.repository.NodeRef;
+import org.alfresco.service.cmr.security.AuthorityService;
+import org.alfresco.service.cmr.security.AuthorityType;
+import org.alfresco.service.cmr.security.PersonService;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.springframework.extensions.webscripts.*;
+
+import java.io.IOException;
+import java.util.*;
+
+public class Members extends AbstractWebScript {
```

Comment:
"Members" is a domain object name. Name of a class should define what it is. "MembersWebScript" would be much better